README.md

```diff
@@ -1 +1,62 @@
-# Math-Magicians
\ No newline at end of file
+## Project Name
+
+# Math-Magicians
+
+# PR
+not yet
+## Built With
+
+- Html
+- css
+- JavaScript with React.js
+
+## live demo :
+-not yet
+
+## Getting Started
+
+-not yet
+
+### Setup
+
+-click the code button in the repo and then download the files as a zip file
+
+#### To get a local copy up and running follow these simple example steps.
+
+- Click the green `Code` button on the repo and copy link
+- In your local PC, open your terminal in the folder you would like to clone the project.
+- Clone the repo with the command: `git clone (copied link)`; like so: `git clone https://github.com/MeqdamAlqudah/Leaderboard`
+- On the terminal, navigate into the directory like so: `cd Leaderboard`
```

Comment:
- [ ] Kindly fix your typo error here. Your documentation refers to **`Leaderboard`** whereas you are working on a different project.